src/server/bootstrap.ts

```diff
@@ -135,6 +148,12 @@ export async function bootstrap() {
       }
     }
 
+    // instantiate pool once and pass as context
```

Comment:
If you separate it on another module ```export const pool``` throws tons of errors.
If you put it at the root of the project resolver, thread instantiation gets stuck.

This is the only way I found it worked. Instantiate it in the express server and pass it as context to the graphql resolvers.